### PR TITLE
Yarn start now works with auto-reloading

### PR DESCRIPTION
**Pull request by michaelfarrell76**
## Related Issues

Updates testing environment to support auto-reloading.

Screen Shot 2022-08-31 at 7 11 30 PM

Screen Shot 2022-08-31 at 7 11 37 PM

https://user-images.githubusercontent.com/10264973/187817548-5a39c22d-c02b-4cfd-b03b-be61411f9447.mov

### CODE REVIEW

File: src/components/AcceptAll.tsx [hunk 1]
```diff
@@ -31,20 +31,24 @@ export default function AcceptAll({
...
-      event.preventDefault();
-      const purposeTypes = airgap.getPurposeTypes();
-      const consent: TrackingConsent = {};
-      Object.keys(purposeTypes).forEach((purpose) => {
-        consent[purpose] = true;
-      });
-      airgap.setConsent(event, consent);
-      handleSetViewState('close');
-    };
+  const handleAcceptAll:
```
Comment (michaelfarrell76) [inline src/components/AcceptAll.tsx:34]: prettier

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "tsc:build": "yarn pnpify tsc --build",
     "clean": "yarn pnpify tsc --build --clean",
     "lint": "yarn pnpify eslint src --ext .ts",
-    "start": "yarn build && yarn live-server --host=localhost --entry-file=./index.html",
+    "start": "yarn build && concurrently 'WATCH=true yarn build' 'yarn live-server --host=localhost --entry-file=./index.html'",
```
Comment (bencmbrook) [inline package.json:41]: `yarn dev`?

Comment (michaelfarrell76) [reply package.json:41]: dont think it really matters here, theres no server, yarn start is more standard than yarn dev imo

File: src/components/AcceptAll.tsx [hunk 3]
```diff
@@ -31,17 +31,20 @@ export default function AcceptAll({
...
+    | JSX.MouseEventHandler<HTMLButtonElement>
+    | undefined = (
```
Comment (bencmbrook) [inline src/components/AcceptAll.tsx:36]: is this confirmed ok w/o event click @eligrey ?

Comment (michaelfarrell76) [reply src/components/AcceptAll.tsx:36]: this didnt change its just prettier, see other comment

**Review by bencmbrook — APPROVED** [review]

File: src/components/AcceptAll.tsx (hunk 3 above)
Comment (eligrey) [reply src/components/AcceptAll.tsx:36]: if event was undefined it would fail higher up with event.preventDefault() anyways so I think this is fine

**Pull request merged**